google-api-client/src/main/java/com/google/api/client/googleapis/services/AbstractGoogleClientRequest.java

```diff
@@ -62,7 +63,8 @@ public abstract class AbstractGoogleClientRequest<T> extends GenericData {
    */
   public static final String USER_AGENT_SUFFIX = "Google-API-Java-Client";
 
-  private static final String API_VERSION_HEADER = "X-Goog-Api-Client";
+  private static final String API_CLIENT_HEADER = "X-Goog-Api-Client";
+  protected static final String API_VERSION_HEADER = "X-Google-Api-Version";
```

Comment:
protected access as this will be used in the generated client classes.

Reply:
Can you add that (the generated code uses this field to set the header) in the source code comment?

Also if there's any public document available for API versions in https://google.aip.dev, reference it from here.